DigitalLearningSolutions.Web/Styles/trackingSystem/viewDelegate.scss

```diff
@@ -23,3 +23,9 @@
     width: 100%;
   }
 }
+
+.nhsuk-summary-list__value {
+  @media (min-width: 40.0625em) {

```

Comment:
Random width? Presumably this is the point where something breaks. Could you clarify what? (and maybe change to px?)

Reply:
This is the point where summary list changes from horizontal to vertical layout.

Reply:
Is that not tied to an NHS breakpoint?

Reply:
Yeah, I'm trying to find it.